packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dashboard/dedicatedCloud.datacenter.dashboard.routes.js

```diff
@@ -8,6 +8,54 @@ export default /* @ngInject */ ($stateProvider) => {
       },
       resolve: {
         breadcrumb: () => null,
+        serviceName: /* @ngInject */ ($transition$) =>
+          $transition$.params().productId,
+        datacenterId: /* @ngInject */ ($transition$) =>
+          $transition$.params().datacenterId,
+        nsxEdgeCurrentLevel: /* @ngInject */ (
+          DedicatedCloud,
+          serviceName,
+          datacenterId,
+        ) => {
+          return DedicatedCloud.getDatacenterInfoNsxt(
+            serviceName,
+            datacenterId,
+          ).then((data) => {
```

Comment:
`data` couldn't be empty ? No risk of error ?

Reply:
A datacenter is always delivered by default with 2 nsx edges, so no, return from this function shouldn't ever be empty